### ISSUE TITLE
Search: prevent tabbing focus from opening the search overlay

### ISSUE THREAD

**Issue by gibrown** [Automattic/jetpack#15923]
> As a sighted keyboard user, assume you want to read the featured posts on your homepage. You tab your way towards this content. Passing through the site search which popups on focus so you hit the Escape key. Now, while the search dialog collapses, it seems as if you can’t proceed tabbing ahead, probably because the search dialog content is only visually hidden but not really hidden in the DOM as should be with visibility hidden or display none.

If I understand correctly, I think there are two things here:
1. Our new opening of the overlay on focus option is triggering the overlay to popup even if the user is trying to tab past it
2. The user can't tab beyond the search box

Maybe fixing the first will also fix the second.

**Comment by bluefuton** [comment]
I think the best solution to this is not to open the overlay on focus. The user should be interacting with a form input positively before the overlay is launched.

Can we offer an alternate way of launching the search experience with a button, so that site owners can provide a direct way into the overlay without hijacking focus on existing elements (perhaps offering that as a sidebar widget)?

**Comment by gibrown** [comment]
https://github.com/Automattic/jetpack/issues/15685 is kinda about documenting and allowing buttons to work better. Background discussion that got us to opening on focus is in https://github.com/Automattic/jetpack/issues/15404 and was specifically about built in buttons in the theme. So not really an easy thing for end users to change or modify.
